### PR TITLE
Add "AI needs AI-ready data" post

### LINKED ISSUE #32: Technical Spice.ai blog post (11/25)

### PR DESCRIPTION
**Pull request by phillipleblanc**
Closes #32 

Also fixes up a previous post

### CODE REVIEW

File: content/posts/project/thinking-in-data.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+### Let's create some AI!
+
+Now let's pretend you are going to create an AI model using traditional techniques. You first need to tell the AI what to learn and then gather the data it will need to actually learn. Once you've done that, you now have the challenge of transforming this raw data into a format that the AI algorithm will be able to use. All AI algorithms can only work on numbers, so you will need to transform any text data to be represented by a number. And the numerical input data usually should be normalized/scaled to have values close to 0.
+
```
Comment (Corentin-pro) [inline content/posts/project/thinking-in-data.md]: I think this would be a good opportunity to explain why and how data should be normalized: values are better if bounded between -1 and 1. Also being to close to 0 would also be a problem because of the lack of precision of floating point representation (usually under 1e-5),

Here's some points about data normalization:

* Neural Network will perform multiplications and additions, starting randomly. This can easily cause 'infinite' values if the input values are too big.
* There are as many possible values between [-1,1] than outside this range. Trying to represent random number above 1 or bellow -1 will have less precision.
* Probability representation: a lot of inputs and outputs are interpreted as probability distributions in machine learning, training mechanisms are commonly using operation such as softmax, tanh, sigmoid, etc that are meant to work in the [-1, 1] range.

File: content/posts/project/thinking-in-data.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
```
Comment (Corentin-pro) [inline content/posts/project/thinking-in-data.md]: I think an illustration to show the 2 path of the flow (training, inference) would be nice here. I like the idea about talking of the difficulties to create and maintain 2 separate code base the should match perfectly.

File: content/posts/project/thinking-in-data.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+### Learn more and contribute
+
+Building intelligent apps that leverage AI is still way too hard, even for advanced developers. Our mission is to make this as easy as creating a modern web page. If the vision resonates with you, join us!
```
Comment (Corentin-pro) [inline content/posts/project/thinking-in-data.md]: I would change the wording of `way too hard` to something tamer like `very complex and daunting`, this may be just me. I feel `too hard` give a sense of impossible or that we can't do but this is common task in the field right now.
